com.unity.cinemachine/Editor/Editors/CinemachineConfiner2DEditor.cs

```diff
@@ -1,22 +1,26 @@
+#if CINEMACHINE_PHYSICS_2D
+
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEditor;
 
-#if CINEMACHINE_PHYSICS_2D
 namespace Cinemachine.Editor
 {
     [CustomEditor(typeof(CinemachineConfiner2D))]
     [CanEditMultipleObjects]
     class CinemachineConfiner2DEditor : BaseEditor<CinemachineConfiner2D>
     {
         SerializedProperty m_MaxWindowSizeProperty;
-        GUIContent m_ComputeSkeletonLabel = new GUIContent(
+        GUIContent m_ComputeSkeletonLabel = new(
             "Oversize Window", "If enabled, the confiner will compute a skeleton polygon to "
                 + "support cases where camera window size is bigger than some regions of the "
                 + "confining polygon.  Enable only if needed, because it's costly");
         GUIContent m_MaxWindowSizeLabel;
-        GUIContent m_InvalidateCacheLabel = new GUIContent(
-            "Invalidate Cache", "Force a recomputation of the polygon cache.  "
+        GUIContent m_InvalidateComputedConfinerLabel = new(
+            "Invalidate Computed Confiner", "Invalidates the current confiner, so a new one is computed next frame.  " +
```

Comment:
I'm still not happy with the names.  Putting myself in the place of the user, I ask: What is the difference between the computed confiner and the cache?  Cache of what?  The help says "invalidate the current confiner".  What is that?

Reply:
It seems to me that there are 2 cached things: the polygon points, and the lens settings.  The names should reflect that, so it's easy to understand which one should be invalidated in what circumstances.